ui/src/views/WorkerManager/WMViewWorkerPools/WMWorkerPools.graphql

```diff
@@ -13,7 +13,6 @@ query workerPools($connection: PageConnection, $filter: JSON) {
         providerId
         description
         owner
-        config
```

Comment:
this was "slightly increasing" worker pools page data size.. instead of 100kb it was fetching 5MB